2-WebApp-graph-user/2-3-Multi-Tenant/Startup.cs

```diff
@@ -78,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
 
             // Sign-in users with the Microsoft identity platform
             services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
-                .AddSignIn("AzureAD", Configuration, options =>
+                .AddSignIn(options =>
                 {
                     Configuration.Bind("AzureAD", options);
 
```

Comment:
We also need to check what happen here. I'm not sure how the overrides of the events will be handled

Reply:
we can sync on this

---
In reply to: [406347523](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/331#discussion_r406347523) [](ancestors = 406347523)